app/src/main/java/com/infomaniak/drive/data/models/UploadFile.kt

```diff
@@ -63,7 +69,9 @@ open class UploadFile(
         private const val DB_NAME = "Sync.realm"
         private const val ONE_DAY = 24 * 60 * 60 * 1000
         private var realmConfiguration: RealmConfiguration = RealmConfiguration.Builder().name(DB_NAME)
+            .schemaVersion(1) // Must be bumped when the schema changes
```

Comment:
I think it's better to put the schema version on a higher line in code (like in a constant) to facilitate the understanding of this codeline ?

Reply:
I think it's a good idea but here we know we're talking about a version so I don't really see the point, except to make it more complex, because if we need to know the current version, we'll have to look for it, whereas here we can see the version directly 😄